main.py

```diff
@@ -1,139 +1,217 @@
-import time
 
+# PIXEL_PIN = board.D5
+# NUM_PIXELS = 20
+
+# pixels = neopixel.NeoPixel(PIXEL_PIN, NUM_PIXELS, brightness=1, auto_write=False)
+
+# steps = 30
+# wait = 0.04
+
+# movingRainbow = 0
+
+
+# def moving_rainbow():    
+#     global movingRainbow
+#     current_mode = read_current_mode()
+#     for r in range(NUM_PIXELS):
+#         if read_current_mode() != current_mode:
+#             return
+#         pixels[r]=t((255/20*(r+movingRainbow))%255)  
+#         pixels.show()
+#     movingRainbow = movingRainbow+1
+
+############################################################################################
+import sys
+print(sys.version)
+import time
 from digitalio import DigitalInOut, Direction, Pull
 import board
 import neopixel
 from rainbowio import colorwheel
 
+
+# Stuff from 2023, might need to be updated
+# connect the pin to D5
```

Comment:
⭐  it is d5, so this comment can go now?

Reply:
👍